process_html.py

```diff
@@ -117,20 +117,19 @@ def main(argv):
             os.makedirs(outputDirectory)
 
         cu.log.setLevel(logging.CRITICAL)
-        json_array, last_bucket = [], str(0)
+	json_array, last_bucket = [], str(0)
 
         fIn = glob.glob( inFolder + '/*/*raw*')
-
         for idx, filename in enumerate(fIn):
 
-            if idx % 10000 == 0:
+            if idx % 1000 == 0:
                 print "Processed %d HTML files" % idx
 
             filenameDetails = filename.split("/")
             urlId = filenameDetails[-1].split('_')[0]
             bucket = filenameDetails[-2]
-
-            if bucket != last_bucket:            
+            
+	    if bucket != last_bucket:            
```

Comment:
Could you add a comment describing the bucket saving troubles we ran into yesterday? i.e. the provided code didn't save the last bucket.